src/app/users/users.component.ts

```diff
@@ -201,15 +201,15 @@ export class UsersComponent implements OnInit, AfterViewInit {
     const myTeamIds = users.map((data) => {
       return data._id || data.doc._id;
     }).concat(userShelf.myTeamIds).reduce(this.dedupeShelfReduce, []);
-    const msg = (myTeamIds.length === 1 ? 'User' : 'Users') + ' added to';
+    const msg = (users.length === 1 ? users[0].name + ' has been' : users.length + ' Users has been') + ' added to';
     this.updateShelf(myTeamIds, userShelf, msg);
   }
 
-  removeTeam(teamId) {
+  removeTeam(teamId, userName) {
     const userShelf = this.userService.getUserShelf();
     const myTeamIds = [ ...userShelf.myTeamIds ];
     myTeamIds.splice(myTeamIds.indexOf(teamId), 1);
-    this.updateShelf(myTeamIds, userShelf, 'User removed from ');
+    this.updateShelf(myTeamIds, userShelf, userName + ' is been removed from');

```

Comment:
Grammar issue, 'is been' => have/has been depending on pluaral/singular or simply *username + ' removed from'*